src/libs/feature-previews-config.ts

```diff
@@ -99,6 +100,14 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
       'Feedback on Azure Collaborative Workflows experience.'
     )}`,
   },
+  {
+    id: ENABLE_WORKFLOWS_SUBMISSION_UX_REVAMP,
+    title: 'Workflows Submission UI Revamp',
+    description: 'Enabling this feature will allow interaction with the updated Submission UI in Azure workspaces',
+    feedbackUrl: `mailto:dsp-sue@broadinstitute.org?subject=${encodeURIComponent(
```

Comment:
If you want to provide an option for feedback, it should be a form or email that goes to the Workflows team, not the SUE team.